tests/conftest.py

```diff
@@ -0,0 +1,7 @@
+import os
+import sys
+
+# It is not preferred to change the python sys.path like this, however it is
+# required since we are importing from the tests folder like:
+# from tests.common.sql_test_case import SqlTestCase
+sys.path = [os.getcwd()] + sys.path
```

Comment:
Some test files contain an import like:

``` python
from tests.common.sql_test_case import SqlTestCase
```

This only works if you run pytest as a module from python, like so:

``` bash
python -m pytest
```

The `pytest` cli command will not work:

``` bash
pytest
```

This is because when you run `python` the current working directory is added to the sys path. This does not happen when you run a command like `pytest`. 

I would recommend using the `pytest` command, since this tests the code in isolation of the current working directory. This is also the command most editors run. However, I did not want to change too much in this PR. Therefore this _trick_ always adds the current working directory to your path, independent of how you run it.